networks/north-america/us/ma/mbta.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
What folder should interstate agencies or multi-state private carriers go into?

Reply:
I figure it'd be similar to the [osm-community-index](https://github.com/osmlab/osm-community-index/tree/master/resources) structure where interstate carriers terminate under the country folder, like `networks/north-america/us/greyhound.json`.

I suppose I already introduced a case where a local agency has a service that crosses state lines with "MBTA Commuter Rail" also reaching Rhode Island. We'll have to decide how strict to be with the structure.